### PR TITLE
Packages cleanups

### PR DESCRIPTION
**Pull request by kulczy**

### CODE REVIEW

**Review by bartek-sek — DISMISSED** [review]

File: src/Sylius/Bundle/AdminBundle/package.json [hunk 1]
```diff
@@ -1,37 +1,38 @@
 {
   "name": "@sylius-ui/admin",
-  "version": "2.0.0-alpha-1",
+  "version": "2.0.0",
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/package.json:3]: Is this version needed?

**Review by GSadee — COMMENTED** [review]
Add a note to the UPGRADE-2.0 file about the changes that need to be made to the end application

File: src/Sylius/Bundle/AdminBundle/package.json (hunk 1 above)
Comment (kulczy) [reply src/Sylius/Bundle/AdminBundle/package.json:3]: nope ;)

Comment (kulczy) [reply src/Sylius/Bundle/AdminBundle/package.json:3]: turns out it is 😁

**Review by mpysiak — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Pull request merged**